index.html

```diff
@@ -98,109 +98,7 @@ <h2 class="header-title green-text">I&apos;m a software developer</h2>
         <h2 class="section-title">My Recent Works</h2>
```

Comment:
- [ ] Please kindly make sure that the mobile version of the modal popup is responsive. Kindly compare to the original design below
- [ ] Also, the space at the bottom is too much when compared to the design. Please kindly remove it

### Your Design
![image](https://user-images.githubusercontent.com/43995830/168033731-16aba32a-2b08-4ddf-99d4-d48664b47368.png)
![image](https://user-images.githubusercontent.com/43995830/168034777-9cb5d332-2158-49c6-9143-efcd2d501a47.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/168034937-edcd1a64-7f19-4917-9e8c-cce11e77f1d7.png)